superset/assets/src/dashboard/stylesheets/dashboard.less

```diff
@@ -38,6 +38,22 @@
   }
 }
 
+.dashboard .dashboard-header {
+  #save-dash-split-button {
+    border-radius: 0;
+    margin-left: -5px;
+    padding: 6px;
```

Comment:
I'm surprised this is 6px, most everything should be a multiple of 8px

Reply:
this padding is not necessary. i removed it.